Code snippets/src/main/java/use_case/quiz/QuizInteractor.java

```diff
@@ -32,11 +32,11 @@ public void execute(QuizInputData quizInputData) {
             selectedQuestions.add(questions.get(i));
             selectedAnswers.add(answers.get(i));
         }
-
-        String[] lastQuestion = chatGPTQuestion(questions, answers);
-        selectedQuestions.add(lastQuestion[0]);
-        selectedAnswers.add(lastQuestion[1]);
-
+        if (!selectedAnswers.isEmpty()) {
+            String[] lastQuestion = chatGPTQuestion(questions, answers);
+            selectedQuestions.add(lastQuestion[0]);
+            selectedAnswers.add(lastQuestion[1]);
+        }
         QuizOutputData quizOutputData = new QuizOutputData(selectedQuestions, selectedAnswers);
```

Comment:
Added the if condition here to facilitate the failureTest: if there are no notes marked as definition or question, there shouldn't be any ChatGPT questions either.